app/Services/QueryServiceProvider.js

```diff
@@ -37,6 +37,60 @@ async function getUnresolvedVocabulary(languagePackageId, userId) {
   return number;
 }
 
+// return the number of unresolved vocabulary
+async function getQueryVocabulary(languagePackageId, userId, limit) {
+  // Get drawers belonging to languagePackage
+  const drawers = await Drawer.findAll({
+    attributes: ['id', 'name', 'queryInterval'],
+    where: {
+      userId: userId,
+      languagePackageId: languagePackageId,
+    },
+  });
+
+  let vocabs = [];
+
+  for (let key = 0; key < drawers.length; ++key) {
```

Comment:
Use a for of loop

Reply:
and add
```js
/* eslint-disable no-await-in-loop */
for (const drawer of Object.values(drawers)) {
   /* ...... */
}
/* eslint-enable no-await-in-loop */
```

Reply:
These comments aren't working unfortunately

Reply:
These work perfektly fine.

Reply:
![image](https://user-images.githubusercontent.com/54618409/112108927-2d468280-8bb1-11eb-843e-e714967c25d8.png)
For me it doesn't work somehow. But if the problem is only for me locally and not in Github Actions it should be ok

Reply:
Try reload vscode (CTRL + SHIFT + P and then search for `Developer: Reload Window`)